src/djcytoscape/tests/test_fields.py

```diff
@@ -8,12 +8,26 @@
 class CytoscapeGFKChoiceFieldTest(TenantTestCase):
 
     def test_queryset(self):
+        """ Quick test to see if the hardcoded model list is equal to CytoScape.ALLOWED_INITIAL_CONTENT_TYPES """
         from djcytoscape.forms import CytoscapeGFKChoiceField
 
-        """ Quick test to see if the hardcoded model list is equal to CytoScape.ALLOWED_INITIAL_CONTENT_TYPES """
         dynamically_loaded_models = [
             ct.model_class() for ct in ContentType.objects.filter(CytoScape.ALLOWED_INITIAL_CONTENT_TYPES)]
 
         f = CytoscapeGFKChoiceField()
 
         self.assertEqual(dynamically_loaded_models, [qs.model for qs in f.queryset.get_querysets()])
+
+    def test_queryset_is_filterable(self):
+        from djcytoscape.forms import CytoscapeGFKChoiceField
```

Comment:
It's not at all clear to me what this test is actually testing.  There's no docstring, the code is condensed and has no comments either.

Reply:
It's a test to see if objects "in use" are being excluded from widget's queryset or not.